ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerview.js

```diff
@@ -62,12 +62,15 @@
         updateValue(value) {
             if (this.widgetObject) {
                 if (this.isActive()) {
-                    this.widgetObject.setValue(value);
+                    // Replacing undefined / null with empty string on reset
+                    this.widgetObject.setValue(value ? value : '');
```

Comment:
this.widgetObject.setValue(value || '');

Reply:
done